spec/models/box_spec.rb

```diff
@@ -326,4 +326,6 @@
       )
     end
   end
-end
\ No newline at end of file
+
+  it { is_expected.to belong_to(:assembly_location).class_name('Location').optional }
```

Comment:
Shoulda also supports to check the `inverse_of` attribute.

Reply:
Nice! thanks!